libs/hashing/include/hpx/hashing/fibhash.hpp

```diff
@@ -29,18 +31,19 @@ namespace hpx { namespace util {
         template <std::size_t N>
         struct hash_helper
         {
-            HPX_STATIC_CONSTEXPR std::size_t log2 =
+            HPX_STATIC_CONSTEXPR std::uint64_t log2 =
```

Comment:
This needs `cstdint`. Should the template parameter be `uint64_t` too?

Reply:
Right, most probably.

Reply:
Done.